### PR TITLE
#1789 - Dependency settings not passed to webserver

### PR DESCRIPTION
**Pull request by JohnPTobe**
##### Checklist

- [x] `manage.py test` passes
- [x] tests are included

### Affected app(s)
- scheduler

### Description of change
Fix for status bugs in webserver.

### CODE REVIEW

**Comment by JohnPTobe** [comment]
need to finish testing these changes in dev

File: dockerfiles/framework/scale/bootstrap.py [hunk 1]
```diff
@@ -286,6 +299,11 @@ def deploy_webserver(client, app_name, es_url, db_url, broker_url):
         'DCOS_PACKAGE_FRAMEWORK_NAME': FRAMEWORK_NAME,
         'DCOS_SERVICE_ACCOUNT': str(secrets_dcos_sa),
         'ENABLE_WEBSERVER': 'true',
+        'FLUENTD_BUFFER_WARN': str(FLUENTD_BUFFER_WARN),
```
Comment (gisjedi) [inline dockerfiles/framework/scale/bootstrap.py]: Any of these new environment variables being added need to be documented in the table at the bottom of the `scale/README.md` file. I certainly have no idea what the difference is between BUFFER_WARN and BUFFER_SIZE_WARN... We need some explanation of what they mean.

**Comment by JohnPTobe** [comment]
Docs updated; issue created 👍

File: scale/README.md [hunk 2]
```diff
@@ -242,15 +242,19 @@ below for reference.
 | ENABLE_BOOTSTRAP            | 'true'                          | Bootstrap Scale support containers         |
 | ENABLE_WEBSERVER            | 'true' or None                  | Used by bootstrap to enable UI and API     |
 | FLUENTD_DOCKER_IMAGE        | 'geoint/scale-fluentd'          | Docker image for fluentd                   |
+| FLUENTD_BUFFER_WARN         | 10                              | Warn if buffer exceeds this many items     |
```
Comment (emimaesmith) [inline scale/README.md]: Could we get these alphabetized? `FLUENTD_BUFFER_*` should go before `FLUENTD_DOCKER_IMAGE` and `MESSSAGE_QUEUE_DEPTH_WARN` after `MESOS_ROLE`.

Comment (JohnPTobe) [reply scale/README.md]: 👍

**Pull request merged**